src/main/java/com/mraof/minestuck/item/MSFoods.java

```diff
@@ -31,6 +32,8 @@ public class MSFoods
     public static final FoodProperties UNKNOWABLE_EGG = (new FoodProperties.Builder()).nutrition(3).saturationMod(0.3F).build();
     public static final FoodProperties BREADCRUMBS = (new FoodProperties.Builder()).nutrition(1).saturationMod(0.1F).fast().build();
     public static final FoodProperties SALAD = new FoodProperties.Builder().nutrition(1).saturationMod(0.6F).build();
+    public static final FoodProperties SOPOR_SLIME_PIE = (new FoodProperties.Builder()).nutrition(1).saturationMod(0.0F).effect(() -> new MobEffectInstance(MSEffects.SOPOR_SICKNESS.get(), 3000, 0), 1.0F).build();
```

Comment:
How would you feel about the effect duration being about half of what it is now?